From 9f46e83e1ac8cef33fe4acbd33c4b91e4017c5d7 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Mon, 16 Apr 2018 01:19:34 +0300
Subject: [PATCH 3/4] recovery: Don't try to set ro.adb.secure

* This is a default property which is blocked by SELinux
  when we try to write it
* We don't need to write it anyway because
  it's always true on usedebug & user builds

Change-Id: Ib398e8fb5749b13014cb43b74e4e431591d978b5
---
 recovery.cpp | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/recovery.cpp b/recovery.cpp
index d27ff8c9..e90d28de 100644
--- a/recovery.cpp
+++ b/recovery.cpp
@@ -1534,19 +1534,13 @@ static void set_time() {
 }
 
 static void setup_adbd() {
-    bool adb_keys_root_exists = false;
     int tries;
     for (tries = 0; tries < 5; ++tries) {
         if (access(adb_keys_root, F_OK) == 0) {
-            adb_keys_root_exists = true;
             break;
         }
         sleep(1);
     }
-    if (adb_keys_root_exists) {
-        // Enable secure adbd
-        property_set("ro.adb.secure", "1");
-    }
 
     // Trigger (re)start of adb daemon
     property_set("service.adb.root", "1");
-- 
2.15.1

